py3-tfx-bsl.yaml

```diff
@@ -0,0 +1,57 @@
+package:
```

Comment:
This looks good (all things considered) and will allow us to at least move forward. However as support is limited to x86 and tfx doesn't provide a package that is compatible with > python 3.10 - this might be better kept as a private package until the above is resolved

Reply:
I have a feeling that image builds will fail if there's not apks for both aarch64 and x86.  @imjasonh is that right?